judgments/views.py

```diff
@@ -65,19 +65,27 @@ def browse(request, court=None, subdivision=None, year=None):
 def detail(request, judgment_uri):
     context = {}
     try:
-        results = api_client.eval_xslt(judgment_uri)
-        xml_results = api_client.get_judgment_xml(judgment_uri)
-        multipart_data = decoder.MultipartDecoder.from_response(results)
-        judgment = multipart_data.parts[0].text
-        model = Judgment.create_from_string(xml_results)
-        context["judgment"] = judgment
-        context["page_title"] = model.metadata_name
-        context["judgment_uri"] = judgment_uri
-        context["pdf_size"] = get_pdf_size(judgment_uri)
-    except MarklogicResourceNotFoundError:
+        is_published = api_client.get_published(judgment_uri)
+    except MarklogicAPIError:
         raise Http404("Judgment was not found")
-    template = loader.get_template("judgment/detail.html")
-    return TemplateResponse(request, template, context={"context": context})
+
+    if is_published:
+        try:
+            results = api_client.eval_xslt(judgment_uri)
+            xml_results = api_client.get_judgment_xml(judgment_uri)
+            multipart_data = decoder.MultipartDecoder.from_response(results)
+            judgment = multipart_data.parts[0].text
+            model = Judgment.create_from_string(xml_results)
+            context["judgment"] = judgment
+            context["page_title"] = model.metadata_name
+            context["judgment_uri"] = judgment_uri
+            context["pdf_size"] = get_pdf_size(judgment_uri)
+        except MarklogicResourceNotFoundError:
+            raise Http404("Judgment was not found")
+        template = loader.get_template("judgment/detail.html")
+        return TemplateResponse(request, template, context={"context": context})
+    else:
+        raise Http404("This Judgment is not available")
```

Comment:
There's a part of me going "shouldn't this be a 403?" but I think it's conceptually simpler for consumers iterating over all the numbers to get a 404.